devtools/src/init-debug.js

```diff
@@ -0,0 +1,20 @@
+// This file is shared between the extension and the Arcs Runtime.
+'use strict';
+
+export default function initDebug() {
+  let root = typeof window === 'object' ? window : global;
+  let arcList = root._arcDebugHandles || [];
+  for (let arc of arcList) {
+    arc.initDebug();
+  };
+  root._arcDebugHandles = {
```

Comment:
instead of storing this as a global, you should be able to export it from this module and import at the usage sites.

Reply:
Everything is encapsulated now in /devtools/shared/arc-registry.js, which I think is indeed much nicer.